build/MSBuildExtensions.targets

```diff
@@ -142,6 +142,46 @@
               PropertyName="_UseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion" />
     </GetUseBundledNETCoreAppPackageVersionAsDefaultNetCorePatchVersion>
 
+    <ItemGroup>
+      <ImplicitPackageVariable Include="Microsoft.NETCore.App"
```

Comment:
We are going to delete the "old" code for this from this file when merging the SDK back here?

Reply:
Yes, I think we should do that.